### ISSUE TITLE
Can't resolve './template'

### ISSUE THREAD

**Issue by josemarluedke** [embroider-build/embroider#124]
Now that babel 7 works, I was able to start testing embroider with some internal apps and addons. 

I found this problem while using one addon that uses pods internally for its components.

The addon has the following component tree: 

```
addon
├── components
│   ├── accordion-section
│   │   ├── component.js
│   │   └── template.hbs
```

Where the `template.hbs` is being imported from `component.js`. 

Embroider fails with the following error: `Module not found: Error: Can't resolve './template' in '/private/var/folders/67/6jb5flcd6d30lkgvj0_92c_m0000gn/T/embroider/0afb4d/node_modules/@neighborly/ui/components/accordion-section'`. 
See full error below.

I have a reproduction here: https://github.com/josemarluedke/my-embroider-test/tree/addon-pods

Using Embroider v0.3.3

```
=================================================================================

ENV Summary:

  TIME: Wed Apr 10 2019 21:02:40 GMT-0700 (Pacific Daylight Time)
  TITLE: ember
  ARGV:
  - /Users/josemarluedke/.nodenv/versions/11.12.0/bin/node
  - /Users/josemarluedke/.nodenv/versions/11.12.0/bin/ember
  - s
  - -p
  - 5200
  EXEC_PATH: /Users/josemarluedke/.nodenv/versions/11.12.0/bin/node
  TMPDIR: /var/folders/67/6jb5flcd6d30lkgvj0_92c_m0000gn/T
  SHELL: /bin/zsh
  PATH:
  - /Users/josemarluedke/.nodenv/versions/11.12.0/bin
  - /usr/local/Cellar/nodenv/1.2.0/libexec
  - /Users/josemarluedke/.rbenv/shims
  - /Users/josemarluedke/.nodenv/shims
  - /usr/local/bin
  - /usr/bin
  - /bin
  - /usr/sbin
  - /sbin
  - /Library/TeX/texbin
  - /usr/local/MacGPG2/bin
  - /Users/josemarluedke/code/go/bin
  - /Users/josemarluedke/.fzf/bin
  PLATFORM: darwin x64
  FREEMEM: 447352832
  TOTALMEM: 17179869184
  UPTIME: 640287
  LOADAVG: 1.89208984375,2.0126953125,2.04736328125
  CPUS:
  - Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz - 2900
  - Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz - 2900
  - Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz - 2900
  - Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz - 2900
  - Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz - 2900
  - Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz - 2900
  - Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz - 2900
  - Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz - 2900
  ENDIANNESS: LE
  VERSIONS:
  - ares: 1.15.0
  - brotli: 1.0.7
  - cldr: 34.0
  - http_parser: 2.8.0
  - icu: 63.1
  - llhttp: 1.1.1
  - modules: 67
  - napi: 4
  - nghttp2: 1.34.0
  - node: 11.12.0
  - openssl: 1.1.1b
  - tz: 2018e
  - unicode: 11.0
  - uv: 1.26.0
  - v8: 7.0.276.38-node.18
  - zlib: 1.2.11

ERROR Summary:

  - broccoliBuilderErrorStack: ModuleNotFoundError: Module not found: Error: Can't resolve './template' in '/private/var/folders/67/6jb5flcd6d30lkgvj0_92c_m0000gn/T/embroider/0afb4d/node_modules/@neighborly/ui/components/accordion-section'
    at factory.create (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/webpack/lib/Compilation.js:821:10)
    at factory (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/webpack/lib/NormalModuleFactory.js:397:22)
    at resolver (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/webpack/lib/NormalModuleFactory.js:130:21)
    at asyncLib.parallel (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/webpack/lib/NormalModuleFactory.js:224:22)
    at /Users/josemarluedke/code/tmp/my-embroider-test/node_modules/neo-async/async.js:2825:7
    at /Users/josemarluedke/code/tmp/my-embroider-test/node_modules/neo-async/async.js:6886:13
    at normalResolver.resolve (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/webpack/lib/NormalModuleFactory.js:214:25)
    at doResolve (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/enhanced-resolve/lib/Resolver.js:184:12)
    at hook.callAsync (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:15:1)
  - code: [undefined]
  - codeFrame: Module not found: Error: Can't resolve './template' in '/private/var/folders/67/6jb5flcd6d30lkgvj0_92c_m0000gn/T/embroider/0afb4d/node_modules/@neighborly/ui/components/accordion-section'
  - errorMessage: Module not found: Error: Can't resolve './template' in '/private/var/folders/67/6jb5flcd6d30lkgvj0_92c_m0000gn/T/embroider/0afb4d/node_modules/@neighborly/ui/components/accordion-section'
        at PackagerRunner (@embroider/webpack)
-~- created here: -~-
    at new Plugin (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/broccoli-plugin/index.js:7:31)
    at new PackagerRunner (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/@embroider/core/src/to-broccoli-plugin.js:11:13)
    at Object.defaultPipeline [as compatBuild] (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/@embroider/compat/src/default-pipeline.js:30:12)
    at module.exports (/Users/josemarluedke/code/tmp/my-embroider-test/ember-cli-build.js:24:39)
    at Builder.readBuildFile (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/ember-cli/lib/models/builder.js:52:14)
    at Builder.setupBroccoliBuilder (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/ember-cli/lib/models/builder.js:66:22)
    at new Builder (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/ember-cli/lib/models/builder.js:32:10)
    at ServeTask.run (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/ember-cli/lib/tasks/serve.js:45:55)
    at Promise.resolve.then (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/ember-cli/lib/models/command.js:243:46)
    at tryCatcher (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:323:19)
-~- (end) -~-
  - errorType: Build Error
  - location:
    - column: [undefined]
    - file: [undefined]
    - line: [undefined]
    - treeDir: [undefined]
  - message: Module not found: Error: Can't resolve './template' in '/private/var/folders/67/6jb5flcd6d30lkgvj0_92c_m0000gn/T/embroider/0afb4d/node_modules/@neighborly/ui/components/accordion-section'
        at PackagerRunner (@embroider/webpack)
-~- created here: -~-
    at new Plugin (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/broccoli-plugin/index.js:7:31)
    at new PackagerRunner (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/@embroider/core/src/to-broccoli-plugin.js:11:13)
    at Object.defaultPipeline [as compatBuild] (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/@embroider/compat/src/default-pipeline.js:30:12)
    at module.exports (/Users/josemarluedke/code/tmp/my-embroider-test/ember-cli-build.js:24:39)
    at Builder.readBuildFile (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/ember-cli/lib/models/builder.js:52:14)
    at Builder.setupBroccoliBuilder (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/ember-cli/lib/models/builder.js:66:22)
    at new Builder (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/ember-cli/lib/models/builder.js:32:10)
    at ServeTask.run (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/ember-cli/lib/tasks/serve.js:45:55)
    at Promise.resolve.then (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/ember-cli/lib/models/command.js:243:46)
    at tryCatcher (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:323:19)
-~- (end) -~-
  - name: BuildError
  - nodeAnnotation: @embroider/webpack
  - nodeName: PackagerRunner
  - originalErrorMessage: Module not found: Error: Can't resolve './template' in '/private/var/folders/67/6jb5flcd6d30lkgvj0_92c_m0000gn/T/embroider/0afb4d/node_modules/@neighborly/ui/components/accordion-section'
  - stack: ModuleNotFoundError: Module not found: Error: Can't resolve './template' in '/private/var/folders/67/6jb5flcd6d30lkgvj0_92c_m0000gn/T/embroider/0afb4d/node_modules/@neighborly/ui/components/accordion-section'
    at factory.create (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/webpack/lib/Compilation.js:821:10)
    at factory (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/webpack/lib/NormalModuleFactory.js:397:22)
    at resolver (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/webpack/lib/NormalModuleFactory.js:130:21)
    at asyncLib.parallel (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/webpack/lib/NormalModuleFactory.js:224:22)
    at /Users/josemarluedke/code/tmp/my-embroider-test/node_modules/neo-async/async.js:2825:7
    at /Users/josemarluedke/code/tmp/my-embroider-test/node_modules/neo-async/async.js:6886:13
    at normalResolver.resolve (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/webpack/lib/NormalModuleFactory.js:214:25)
    at doResolve (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/enhanced-resolve/lib/Resolver.js:184:12)
    at hook.callAsync (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/Users/josemarluedke/code/tmp/my-embroider-test/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:15:1)

=================================================================================
```

**Comment by ef4** [comment]
Thanks for testing. I think this is the same as #10.

I'm looking into a fix this week.

**Comment by ef4** [comment]
I think this is fixed in 0.3.4.
